cfgov/unprocessed/css/post.less

```diff
@@ -373,31 +373,6 @@
     - cfgov-post
 */
 
-.post_heading {
```

Comment:
Yeah, the sections of the browser tests referencing these classes can be deleted. We are trying to move away from testing dom element references and towards focusing on functionality.